web/src/main/java/org/springframework/security/web/util/matcher/AntPathRequestMatcher.java

```diff
@@ -67,6 +67,17 @@ public final class AntPathRequestMatcher implements RequestMatcher, RequestVaria
 
 	private final UrlPathHelper urlPathHelper;
 
+	/**
+	 * Creates a matcher that will match all request with the supplied HTTP method in a
+	 * case-sensitive manner.
+	 * @param method the HTTP method. The {@code matches} method will return false if the
+	 * incoming request doesn't have the same method.
+	 */
+	public static AntPathRequestMatcher forMethod(HttpMethod method) {
+		Assert.notNull(method, "method cannot be null");
+		return new AntPathRequestMatcher(MATCH_ALL, method.name());
+	}
+
```

Comment:
There doesn't seem to be a way to specify a pattern or a method and pattern via static factory method